README.md

```diff
@@ -35,19 +35,28 @@ Contains the JSON schemas used to validate the sources in the integration tests.
+|Field Name|Type|Presence|Definition|                                                                     
+|-|-|-|-|
+| mdb_source_id |  Unique ID | System generated | Unique numerical identifier.      |
+| data_type     | Enum| Required| The data format that the source uses: `gtfs`.|
+|location| Object | Required |Contains  <ul><li>Text that describes the source's location in the `country_code`, `subdivision_name`, and `municipality` fields.</li><li>Latitude, longitude, date and time that describes the source's bounding box in the `bounding_box` subobject. </li></ul>|
+| - country_code       | Text |Required                  | ISO 3166-1 alpha-2 code designating the country where the system is located. For a list of valid codes [see here](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes).                                                    |
+| - subdivision_name  | Text |Optional              | ISO 3166-2 subdivision name designating the subdivision (e.g province, state, region) where the system is located. For a list of valid names [see here](https://unece.org/trade/uncefact/unlocode-country-subdivisions-iso-3166-2).|    
+| - municipality  | Text |Optional              | Primary municipality in which the transit system is located.|         
+| - bounding_box  | Object|System generated             | Bounding box of the data source when it was first added to the catalog. Contains `minimum_latitude`, `maximum_latitude`, `minimum_longitude`, `maximum_longitude` and `extracted_on` fields. If the bounding box information displays as "null", you can check any potential source errors with [the GTFS validator](https://github.com/MobilityData/gtfs-validator).   |  
+| --minimum_latitude    | Latitude | System generated                    | The minimum latitude for the source's bounding box.     
+| --maximum_latitude    | Latitude | System generated                    | The maximum latitude for the source's bounding box.   
+| --minimum_longitude    | Longitude | System generated                    | The minimum longitude for the source's bounding box.    
+| --maximum_longitude    | Longitude | System generated                    | The maximum longitude for the source's bounding box.  
+| --extracted_on   | Date and Time | System generated                    | The date and timestamp the bounding box was extracted on in UTC.                                                                         
+| provider     | Text | Required                   | A commonly used name for the transit provider included in the source.  
+| name        |  Text |Optional              | An optional description of the data source, e.g to specify if the data source is an aggregate of multiple providers, or which network is represented by the source. |
+|URLs| Object | Required | Contains URLs associated with the source in the `direct_download_url`, `latest`, and `license` fields.
```

Comment:
Since every other field name is represented by its key in the JSON schema, I would do the same for `urls`. I believe it should also be done for the GTFS Realtime data structure below.
```suggestion
|urls| Object | Required | Contains URLs associated with the source in the `direct_download_url`, `latest`, and `license` fields.
```